src/plone/restapi/services/registry/get.py

```diff
@@ -35,5 +38,20 @@ def reply(self):
             value = registry[self._get_record_name]
             return json_compatible(value)
         else:  # batched listing
-            serializer = getMultiAdapter((registry, self.request), ISerializeToJson)
+            if q := self.request.form.get("q"):
+                # Disable CSRF protection
+                if "IDisableCSRFProtection" in dir(plone.protect.interfaces):
+                    alsoProvides(
+                        self.request, plone.protect.interfaces.IDisableCSRFProtection
+                    )
```

Comment:
```suggestion
```

The CSRF protection shouldn't be needed for a GET request that doesn't write anything to the database. If this _does_ write something to the database, that's a problem that we need to know about, not ignore.